marketplace_builder/views/pages/reference/form-configurations/form-configurations.liquid

```diff
@@ -9,6 +9,26 @@ searchable: true
 
 **Form Configurations** give you full control over both the look and feel of a form and its behavior.
 
+## Configuration Properties
+
+| Property                 | Description | Options | Default | Required |
+| ------------------------ | ----------- | ------- | ------- | -------- |
+| `name`                   |             |         |         | Yes      |
+| `resource`               |             |         |         | Yes      |
+| `api_call_notifications` |             |         |         | No       |
+| `async_callback_actions` |             |         |         | No       |
+| `authorization_policies` |             |         |         | No       |
+| `callback_actions`       |             |         |         | No       |
+| `configuration`          |             |         |         | No       |
+| `default_payload`        |             |         |         | No       |
+| `email_notifications`    |             |         |         | No       |
+| `flash_alert`            |             |         |         | No       |
+| `flash_notice`           |             |         |         | No       |
+| `redirect_to`            |             |         |         | No       |
+| `resource_owner`         |             |         | `self`  | No       |
+| `sms_notifications`      |             |         |         | No       |
+| `spam_protection`        |             |         |         | No       |
```

Comment:
@Slashek spam_protection isnt supported anymore, right?

Reply:
it is - recaptcha :)